panther_driver/src/battery_node.py

```diff
@@ -2,153 +2,128 @@
 
 import math
 import yaml
-from numpy import NaN
 
 import rospy
-from sensor_msgs.msg import BatteryState
-from panther_msgs.msg import BatteryDriver
-
 
-class BatteryNode:
-    def __init__(self, name):
-        rospy.init_node(name)
+from sensor_msgs.msg import BatteryState
+from panther_msgs.msg import DriverState
```

Comment:
```suggestion
from sensor_msgs.msg import BatteryState

from panther_msgs.msg import DriverState
```

According to our coding style. There should be a blank line between those imports